addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -875,7 +913,45 @@ ListRenderer.include({
     _onNavigationMove: function (ev) {
         var self = this;
         ev.stopPropagation(); // stop the event, the action is done by this renderer
+        var currentRowID, currentWidget, currentRecord;
+        var moveUpDown = false;
+        var self = this;
+        if (this.currentRow !== null) {
+            currentRecord = this.state.data[this.currentRow]
+            currentRowID = currentRecord.id;
+            currentWidget = this.allFieldWidgets[currentRowID][this.currentFieldIndex];
+            if (
+            currentWidget &&
+            currentWidget.field &&
+            currentWidget.field.type !== "many2one" &&
+            currentWidget.field.type !== "many2many" &&
+            currentWidget.field.type !== "selection" &&
+            currentWidget.field.type !== "text" //TODO other ones?
```

Comment:
wow that's ugly... What's the idea here ?

Reply:
Thanks for the remark, you're right it was not needed because the propagation of the event is stopped in this case. I removed it and it still works